test/promise.cpp

```diff
@@ -82,7 +82,7 @@ class local_static_allocator {
 
   template<typename _T>
   struct rebind {
-    using other = local_static_allocator<_T, MaxSize, alignment>;
+    typedef local_static_allocator<_T, MaxSize, alignment> other;
```

Comment:
no using is preferable way to create an alias.

Reply:
That was what I did initially. This was my desparate attempt to make windows work :-P